core/src/main/java/tachyon/master/MasterInfo.java

```diff
@@ -2133,6 +2135,21 @@ public boolean rename(TachyonURI srcPath, TachyonURI dstPath) throws FileDoesNot
   }
 
   /**
+   * Rename a file. This method is supposed to be called in unittest only.
+   *
+   * @param fileId The id of the file to rename
+   * @param dstPath The new path of the file
+   * @param opTimeMs The time of the rename operation, in milliseconds
+   * @return true if the rename succeeded, false otherwise
+   * @throws FileDoesNotExistException If the id doesn't point to an inode
+   * @throws InvalidPathException if the source path is a prefix of the destination
+   */
+  public boolean rename(int fileId, TachyonURI dstPath, long opTimeMs)
```

Comment:
This wrapper method is removed, as the unittest now directly calls renameInternal now.